DQM/TrackingMonitorClient/data/tracking_qualitytest_config_tier0.xml

```diff
@@ -17,29 +17,50 @@
      <TYPE>ContentsXRange</TYPE> 
      <PARAM name="error">0.80</PARAM> 
      <PARAM name="warning">0.90</PARAM> 
-     <PARAM name="xmin">5.0</PARAM> 
+     <PARAM name="xmin">3.0</PARAM> 
      <PARAM name="xmax">35.0</PARAM> 
 </QTEST>
-<QTEST name="MeanWithinExpected:NumberOfRecHitsPerTrack" activate="true"> 
+<QTEST name="MeanWithinExpected:NumberOfRecHitsPerTrack" activate="true">
+     <TYPE>MeanWithinExpected</TYPE> 
+     <PARAM name="error">0.80</PARAM> 
+     <PARAM name="warning">0.90</PARAM> 
+     <PARAM name="mean">1.0</PARAM>
+     <PARAM name="useRMS">0</PARAM>
+     <PARAM name="useSigma">0</PARAM>
+     <PARAM name="useRange">1</PARAM>
+     <PARAM name="minEntries">0</PARAM>
+     <PARAM name="xmin">8.0</PARAM>
+     <PARAM name="xmax">15.0</PARAM>  
+     <!--
      <TYPE>ContentsXRange</TYPE> 
      <PARAM name="error">0.80</PARAM> 
      <PARAM name="warning">0.90</PARAM> 
      <PARAM name="xmin">8.0</PARAM> 
-     <PARAM name="xmax">15.0</PARAM> 
+     <PARAM name="xmax">15.0</PARAM> --> 
 </QTEST>
-<QTEST name="XrangeWithin:NumberOfRecHitsPerTrack" activate="true"> 
+<!--<QTEST name="XrangeWithin:NumberOfRecHitsPerTrack" activate="true"> 
      <TYPE>ContentsXRange</TYPE> 
      <PARAM name="error">0.80</PARAM> 
      <PARAM name="warning">0.90</PARAM> 
-     <PARAM name="xmin">5.0</PARAM> 
+     <PARAM name="xmin">3.0</PARAM> 
      <PARAM name="xmax">35.0</PARAM> 
-</QTEST>
-<QTEST name="MeanWithinExpected:Chi2oNDF" activate="true"> 
-     <TYPE>ContentsXRange</TYPE> 
+</QTEST>-->
+<QTEST name="MeanWithinExpected:Chi2oNDF" activate="true">
+     <TYPE>MeanWithinExpected</TYPE> 
+     <PARAM name="error">0.85</PARAM> 
+     <PARAM name="warning">0.95</PARAM> 
+     <PARAM name="mean">1.0</PARAM>
+     <PARAM name="useRMS">0</PARAM>
+     <PARAM name="useSigma">0</PARAM>
+     <PARAM name="useRange">1</PARAM>
+     <PARAM name="minEntries">0</PARAM>
+     <PARAM name="xmin">0.5</PARAM>
+     <PARAM name="xmax">1.5.0</PARAM> 
```

Comment:
Hi Tomas,
you are right, it is a typo. Anyway the test run in the expected way, I 
will close the PR, correct and open again.
Thanks for spotting!!


Il 16/09/2015 22.37, threus ha scritto:
>
> In 
> DQM/TrackingMonitorClient/data/tracking_qualitytest_config_tier0.xml 
> <https://github.com/cms-sw/cmssw/pull/11292#discussion_r39681763>:
>
> >       <PARAM name="xmax">35.0</PARAM>
> > -</QTEST>
> > -<QTEST name="MeanWithinExpected:Chi2oNDF" activate="true">
> > -     <TYPE>ContentsXRange</TYPE>
> > +</QTEST>-->
> > +<QTEST name="MeanWithinExpected:Chi2oNDF" activate="true">
> > +     <TYPE>MeanWithinExpected</TYPE>
> > +     <PARAM name="error">0.85</PARAM>
> > +     <PARAM name="warning">0.95</PARAM>
> > +     <PARAM name="mean">1.0</PARAM>
> > +     <PARAM name="useRMS">0</PARAM>
> > +     <PARAM name="useSigma">0</PARAM>
> > +     <PARAM name="useRange">1</PARAM>
> > +     <PARAM name="minEntries">0</PARAM>
> > +     <PARAM name="xmin">0.5</PARAM>
> > +     <PARAM name="xmax">1.5.0</PARAM>
>
> probably a typo for xmax here?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/11292/files#r39681763>.
>



---
Questa e-mail è stata controllata per individuare virus con Avast antivirus.
https://www.avast.com/antivirus